tema5/src/main/java/ro/scoalainformala/Main.java

```diff
@@ -0,0 +1,73 @@
+package ro.scoalainformala;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        Person person1 =new Person(34, "John Doe");
```

Comment:
pls try to format the code before committing it 
```suggestion
        Person person1 = new Person(34, "John Doe");
```